test/test_message_text_index_serialization.py

```diff
@@ -3,19 +3,23 @@
 
 """Test for MessageTextIndex serialization to ensure it's no longer a no-op."""
 
-import pytest
 import sqlite3
 
 import numpy as np
+import pytest
```

Comment:
The 3rd party modules should be in one block. But fixtures is local and needs to be separated by a blank line.